client/gutenberg/extensions/publicize/settings-button.jsx

```diff
@@ -25,15 +25,28 @@ import { Component } from '@wordpress/element';
  * Internal dependencies
  */
 import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+import getGutenbergSiteFragment from 'gutenberg/extensions/presets/jetpack/editor-shared/get-gutenberg-site-fragment';
```

Comment:
Ok to have this import, but just noting that SDK bundles everything in `gutenberg/extensions/presets/jetpack/editor-shared/*`:

https://github.com/Automattic/wp-calypso/blob/ad75f7fa3c73c4c9fa984f23af3f7d1185b67ded/bin/sdk/gutenberg.js#L63-L68